docs/source/user_guide/options.rst

```diff
@@ -270,6 +273,5 @@ plotting.sample_ratio           None           'plotting.sample_ratio' sets the
                                                'plotting.max_rows' option.
 plotting.backend                'plotly'       Backend to use for plotting. Default is plotly.
                                                Supports any package that has a top-level `.plot`
-                                               method. Some options are: [matplotlib, plotly,
```

Comment:
I double checked this because I didn't change a line, but it looks like a previous change in config.py wasn't synced. I figured I could add it in this PR as well.

Reply:
Yeah, sounds good!